addons/hr_recruitment/tests/test_recruitment_process.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+from odoo.modules.module import get_module_resource
+
+
+class HrRecruitmentTestCase(common.TransactionCase):
+
+    def setUp(self):
+        super(HrRecruitmentTestCase, self).setUp()
+        self.User = self.env['res.users']
+        self.Mail = self.env['mail.thread']
+        self.HR = self.env['hr.applicant']
+        self.Attachment = self.env['ir.attachment']
+        self.company_id = self.ref('base.main_company')
+        self.group_hr_recruitment_user = self.ref('hr_recruitment.group_hr_recruitment_user')
+        self.job_id = self.ref('hr.job_developer')
+        self.job_stage = self.ref('hr_recruitment.stage_job1')
+
+    def test_00_res_users_hr_recruitment_officer_flow(self):
+        self.res_users_hr_recruitment_officer = self.User.with_context({'res_users_hr_recruitment_officer': True}).create({
```

Comment:
The dictionary literal seems unnecessary, and possibly detrimental. Why not just use a keyword parameter?